etc/config.sample.toml

```diff
@@ -86,6 +86,34 @@ reporting-disabled = false
   # query-log-enabled = true
 
 ###
+### [hinted-handoff]
+###
+### Controls the hinted handoff feature, which allows nodes to temporarily
+### store queued data when one node of a cluster is down for a short period
+### of time.
+###
+
+[hinted-handoff]
+  enabled = true
+  dir = "/var/opt/influxdb/hh"
+  max-size = 1073741824
+  max-age = "168h"
+  retry-rate-limit = 0
+
+  # Hinted handoff will start retrying writes to down nodes at a rate of once per second.
+  # If any error occurs, it will backoff in an exponential manner, until the interval
+  # reaches retry-max-interval. Once writes to all nodes are successfully completed the
+  # interval will reset to retry-interval.
+  retry-interval = "1s"
+  retry-max-interval = "1m"
+
+  # Interval between running checks for data that should be purged. Data is purged from
+  # hinted-handoff queues for two reasons. 1) The data is older than the max age, or
+  # 2) the target node has been dropped from the server. Data is never dropped until
```

Comment:
Yes I do, that Is a typo.

On Friday, October 23, 2015, Paul Dix <notifications@github.com> wrote:

> In etc/config.sample.toml
> <https://github.com/influxdb/influxdb/pull/4516#discussion_r42859470>:
>
> > +  enabled = true
> > +  dir = "/var/opt/influxdb/hh"
> > +  max-size = 1073741824
> > +  max-age = "168h"
> > +  retry-rate-limit = 0
> > +
> > +  # Hinted handoff will start retrying writes to down nodes at a rate of once per second.
> > +  # If any error occurs, it will backoff in an exponential manner, until the interval
> > +  # reaches retry-max-interval. Once writes to all nodes are successfully completed the
> > +  # interval will reset to retry-interval.
> > +  retry-interval = "1s"
> > +  retry-max-interval = "1m"
> > +
> > +  # Interval between running checks for data that should be purged. Data is purged from
> > +  # hinted-handoff queues for two reasons. 1) The data is older than the max age, or
> > +  # 2) the target node has been dropped from the server. Data is never dropped until
>
> Then I don't understand what "target node has been dropped from the
> server" means. That's "target server has been dropped from the server"? Do
> you actually mean "the target server has been dropped from the cluster"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/influxdb/influxdb/pull/4516/files#r42859470>.
>